src/configuration.ts

```diff
@@ -136,7 +138,15 @@ export function loadConfigurationFromPath(configFilePath: string): IConfiguratio
         return require(configFilePath).tslintConfig;
     } else {
         const resolvedConfigFilePath = resolveConfigurationPath(configFilePath);
-        let configFile: IConfigurationFile = require(resolvedConfigFilePath);
+        let fileContent: string;
+        if (path.extname(resolvedConfigFilePath) === ".json") {
+            fileContent = stripComments(fs.readFileSync(resolvedConfigFilePath).toString());
+        } else {
+            fileContent = require(resolvedConfigFilePath);
+            delete require.cache[resolvedConfigFilePath];
+        }
+
+        let configFile: IConfigurationFile = JSON.parse(fileContent);
```

Comment:
`JSON.parse` only needs to be done if loading a JSON file. When we require a `.js` file, we'll get back an object automatically